typescript.js

```diff
@@ -0,0 +1,408 @@
+        "@typescript-eslint/prefer-interface": "off", // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-interface.md
+        "@typescript-eslint/prefer-literal-enum-member": "warn", // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-literal-enum-member.md
+        "@typescript-eslint/prefer-namespace-keyword": "off", // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-namespace-keyword.md
+        "@typescript-eslint/prefer-nullish-coalescing": [
+            "warn",
+            {
+                ignoreConditionalTests: true,
+                ignoreMixedLogicalExpressions: true,
+            },
+        ], // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-nullish-coalescing.md
+        "@typescript-eslint/prefer-optional-chain": "warn", // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-optional-chain.md
+        "@typescript-eslint/prefer-readonly": "off", // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-readonly.md
+        "@typescript-eslint/prefer-readonly-parameter-types": "off", // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-readonly-parameter-types.md
+        "@typescript-eslint/prefer-reduce-type-parameter": "warn", // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-reduce-type-parameter.md
+        "@typescript-eslint/prefer-regexp-exec": "off", // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-regexp-exec.md
+        "@typescript-eslint/prefer-string-starts-ends-with": "warn", // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-string-starts-ends-with.md
+        "@typescript-eslint/prefer-ts-expect-error": "error", // https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/prefer-ts-expect-error.md
+        "@typescript-eslint/promise-function-async": [
+            "warn",
+            {
+                allowAny: true,
+                allowedPromiseNames: [],
+                checkArrowFunctions: true,
+                checkFunctionDeclarations: true,
+                checkFunctionExpressions: true,
+                checkMethodDeclarations: true,
+            },
+        ], // https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin
+        // TODO: check @timidak
+        // def. needs testing
+        "@typescript-eslint/quotes": ["warn", "backtick", {
```

Comment:
just use the same setting as in our base config `quotes: ["error","single", { "allowTemplateLiterals": true }]`.
Please keep in mind to turn the regular quotes rule off when using ts.

Reply:
same here 😕